fossunited/foss_profiles/doctype/foss_user_profile/foss_user_profile.py

```diff
@@ -4,11 +4,16 @@
 import re
 
 import frappe
+from frappe.exceptions import PermissionError
 from frappe.website.website_generator import WebsiteGenerator
 
 from fossunited.api.profile import is_valid_username
 
 
+class PrivateProfileError(PermissionError):
```

Comment:
nitpicky comment - a docstring would be nice here, something along the lines of "exception raised when a private profile is accessed"

Reply:
Added the docstring